spec/models/room_spec.rb

```diff
@@ -40,4 +41,18 @@
       expect(room.total_occupied).to eq(10)
     end
   end
+
+  context "to csv" do
+    let(:room) { FactoryGirl.create(:room) }
+
+    it "#to_csv send all attributes as csv" do
+      attributes = "id,name,capacity,created_at,updated_at\n"
```

Comment:
Can you use "tab" as separator. I know you can define it in CSV and it is clearer (you don't need to scape names of rooms with ,"